capsules/src/kv_driver.rs

```diff
@@ -0,0 +1,453 @@
+//! KV System
+//!
```

Comment:
Update.

Also, if you want you can put the KV code in its own folder. I don't care either way, up to you.